pkg/netpol/connection/connection.go

```diff
@@ -0,0 +1,17 @@
+package connection
+
+import (
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/np-guard/netpol-analyzer/pkg/netpol/internal/common"
+)
+
+// Connection represents a set of allowed connections between two peers
+type Connection interface {
```

Comment:
It might be awkward that the pkg name and interface name are identical, as it has to be used `connection.Connection` , which is just a duplication of the same word.  
How about renaming the interface type name? for example one option is `AllowedSet`, and then it can be used as `connection.AllowedSet`.
It actually represents a set of allowed connections and not a single connection as implied by the current name.

Reply:
done